RSN_experiments/LearnableCov.py

```diff
@@ -1,201 +0,0 @@
-import torch
-from torch import Tensor
```

Comment:
why this this script removed here?

Reply:
good question. appears to be a mistake i made during rebasing